web/geo/generate-zone-bounding-boxes.js

```diff
@@ -6,15 +6,20 @@ const { getJSON } = require('./utilities');
 const { getZonesJson, saveZonesJson } = require('./files');
 
 let zones = getJSON(path.resolve(__dirname, './world.geojson'));
-if (args.length > 0) {
-  zones = zones.filter((d) => d.properties.zoneName === args[0]);
+
+
+if (args.length <= 0) {
+  console.log('ERROR: Please add a zoneName parameter ("node generate-zone-bounding-boxes.js DE"')
```

Comment:
Missing closing parenthesis in the logged text

Reply:
Good catch!